### ISSUE TITLE
onnx convert error

### ISSUE THREAD

**Issue by BaiCHao12** [om-ai-lab/OmDet#12]
onnx convert error
![image](https://github.com/om-ai-lab/OmDet/assets/86549726/0b05d5bb-060b-412c-8347-aa83e53800c5)

**Comment by hx621** [comment]
@BaiCHao12 Thank you for your question. This issue was caused by torch.where， and we've made some changes to avoid this issue. Let  me know if you have any other problems.

**Comment by BaiCHao12** [comment]
Thank you! but， onnx model  and pytorch model results  are different；In the process，I only converted backbone。

![image](https://github.com/om-ai-lab/OmDet/assets/86549726/60ce0669-bd0f-4633-92ae-226e5bfd7c43)

**Comment by BaiCHao12** [comment]
And，in picture，The parameters boxed in red should also be model-inferred。
![image](https://github.com/om-ai-lab/OmDet/assets/86549726/07a75762-d0f7-48a1-807c-b116a6db1f8f)

**Comment by BaiCHao12** [comment]
Thank you,I've fixed this issue

**Comment by BaiCHao12** [comment]
language_backbone（model text embedding part） convert onnx error
![image](https://github.com/om-ai-lab/OmDet/assets/86549726/b00ef892-5540-4a08-8b14-de5a30eea700)

**Comment by hx621** [comment]
> language_backbone（model text embedding part） convert onnx error ![image](https://private-user-images.githubusercontent.com/86549726/343768355-b00ef892-5540-4a08-8b14-de5a30eea700.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4ODQ1OTYsIm5iZiI6MTcxOTg4NDI5NiwicGF0aCI6Ii84NjU0OTcyNi8zNDM3NjgzNTUtYjAwZWY4OTItNTU0MC00YTA4LThiMTQtZGU1YTMwZWVhNzAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAyVDAxMzgxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5ZGNhNDczYjQ0YzFiN2VjMDMyOWQ3ZDE1NGMyZGQ2YmNiOWI5NThkYmE3MzJhZGJjMWNlYzMxMTU4N2ZjOTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qnt7voLJM7SrN6-nOEoZ5z8_pIA3om0l51eSWLWCc_M)

@BaiCHao12 Due to the label cache mentioned in the paper, the language backbone only needs to be inferred once during model inference. Therefore, our project currently does not support the conversion of ONNX models with language backbones. If you have any other requirements, please let us know.
